.gitignore

```diff
@@ -1,4 +1,5 @@
 # Binaries for programs and plugins
+*.jar
```

Comment:
If you wanted you could put the jar in the `tools` dir like the other programs we install. That also means you could add the download to the `tools` make target.

Reply:
yeah sure ill do that now